docs/relational-databases/wmi-provider-configuration-classes/sqlservice-class/sqlservicetype-property-sqlservice-class.md

```diff
@@ -49,6 +49,11 @@ object.SqlServiceType [= value]
 |*5*|MSSQLServerOLAPService is the [!INCLUDE[ssASnoversion](../../../includes/ssasnoversion-md.md)] service.|  
 |*6*|ReportServer is the [!INCLUDE[ssRSnoversion](../../../includes/ssrsnoversion-md.md)] service.|  
 |*7*|SQLBrowser is the [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)] Browser service.|  
+|*8*|NsService is the [!INCLUDE[ssNoVersion](../../../includes/ssns-md.md)] Notification service.|  

```

Comment:
My editor can't handle this include syntax, so it is untested. It looks like it should work, tho.